third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/BuilderUtil.java

```diff
@@ -120,27 +159,43 @@ else if (uid != "")
         return specList;
     }
 
+    /**
+     * Populate the links for references based on a UID. Looker generates mappings for local context
+     * (file specific) and global context (all the UID references). Searching is done first in the
+     * local context and then in the global context if local context is not found.
+     *
+     * ex. {@link Lookup } would search all the references to find which Lookup file to use
+     * It would parse the text to search ("Lookup")
+     *
+     * Since packages (v1 and v1beta) may contain the same generated java file names, there may be some
+     * conflicts between which link it should be.
```

Comment:
Can you add why Javac doesn't have the problem of ambiguity but this tool via javadoc has trouble finding the exact class?